### ISSUE TITLE
Crash from Tools --> Settings

### ISSUE THREAD

**Issue by mahdi1234** [martinrotter/rssguard#184]
```
RSS Guard
Version: 3.5.6 (built on Linux/x86_64)
Revision: 
Build date: 3/26/18 5:22 PM
Qt: 5.9.4 (compiled against 5.9.4)
```

Go Tools - Settings and it crashes with

```
[rssguard] File-based SQLite database connection 'TtRssServiceRoot' to file '/home/mahdi/.rssguard/data/database/local/database.db' seems to be established. (18/26/03 19:09:21)
  Type: DEBUG
  File: src/miscellaneous/databasefactory.cpp (line 756)
  Function: QSqlDatabase DatabaseFactory::sqliteConnection(const QString&, DatabaseFactory::DesiredType)

[rssguard] Settings of BaseNetworkAccessManager loaded. (18/26/03 19:09:21)
  Type: DEBUG
  File: src/network-web/basenetworkaccessmanager.cpp (line 45)
  Function: virtual void BaseNetworkAccessManager::loadSettings()

[rssguard] Tray icon displayed. (18/26/03 19:09:22)
  Type: DEBUG
  File: src/gui/systemtrayicon.cpp (line 87)
  Function: void SystemTrayIcon::showPrivate()

[rssguard] Destroying SilentNetworkAccessManager instance. (18/26/03 19:09:22)
  Type: DEBUG
  File: src/network-web/silentnetworkaccessmanager.cpp (line 19)
  Function: virtual SilentNetworkAccessManager::~SilentNetworkAccessManager()

[rssguard] Destroying FormAbout instance. (18/26/03 19:09:37)
  Type: DEBUG
  File: src/gui/dialogs/formabout.cpp (line 24)
  Function: virtual FormAbout::~FormAbout()

terminate called after throwing an instance of 'IOException'
Aborted

```

**Comment by martinrotter** [comment]
I managed to partially reproduce the issue. Does it crash immediately after you hit tools -> settings?

**Comment by mahdi1234** [comment]
Yes it crashes instantly after cliking/selecting tools - settings item. Same with latest build.
```
RSS Guard
Version: 3.5.7 (built on Linux/x86_64)
Revision: 7a368e53
Build date: 4/3/18 4:48 PM
Qt: 5.9.4 (compiled against 5.9.4)

```

**Comment by martinrotter** [comment]
@mahdi1234 Can you please try if this is now fixed with 27bdad2f?

**Comment by martinrotter** [comment]
Well, I consider this fixed in latest stable version. It seemed to be the bug with badly set "tab order" which resulted in crash and I reproduced it.
